core/bin/external_node/src/config/mod.rs

```diff
@@ -621,26 +630,31 @@ pub(crate) struct RequiredENConfig {
     /// Port on which the healthcheck REST server is listening.
     pub healthcheck_port: u16,
     /// Address of the Ethereum node API.
-    /// Intentionally private: use getter method as it manages the missing port.
```

Comment:
AFAIU, the underlying issue [was fixed in `jsonrpsee` v0.20](https://github.com/paritytech/jsonrpsee/blob/master/CHANGELOG.md#v0200---2023-08-11).